src/App.tsx

```diff
@@ -23,10 +26,14 @@ const downloadBlob = (blob: Blob) => {
 const isImageFile = (file: File): boolean =>
   file.type === "image/png" || file.type === "image/jpeg";
 
+const isFontFile = (file: File): boolean =>
+  file.name.endsWith(".ttf") || file.name.endsWith(".otf");
```

Comment:
I imagine isImageFile is detecting the mimetype? Can we do the same here? The server *does* verify the actual mimetypes, so if you need to know what the font ones are, you can find them in [the Media class](https://github.com/lovebrew/lovebrew-webserver/blob/dev/0.9.0/lovebrew/media.py#L16). Although I don't know if whatever this is using uses the same mimetype names, unless that's a standardized naming scheme.

Reply:
I tried using mimetypes for the font files but for whatever reason it was just `undefined`, so I went with this approach. This is also preferably a temporary solution just to get the UI working and the converters testable and should be looked into when finalizing the feature.